app/services/schools/advice/heating_control_service.rb

```diff
@@ -93,12 +93,12 @@ def find_meters
         @school.meters
                .active
                .gas
-               .where.not(id: MeterAttribute.where(meter_id: @school.meters.pluck(:id))
+               .where.not(id: MeterAttribute.where(meter_id: @school.meters.pluck(:id), attribute_type: :function_switch)
                                             .where(
```

Comment:
perhaps a separate PR but seems like this would be clearer without the heredoc and indentation of the last bracket seems odd (rubocop advising me to put it on the line above)